src/style/themes.ts

```diff
@@ -81,17 +91,17 @@ export const lightTheme: DefaultTheme = {
   name: "light",
   colors: {
     primary: "#2B71FF",
-    primaryDark: "#0F5FFF",
+    primaryDark: "#101217",
     white: "#060607",
-    alwaysWhite: '#FFF',
-    black: "#FFF",
-    alwaysBlack: "#000",
+    alwaysWhite: "#FFFFFF",
+    black: "#FFFFFF",
+    alwaysBlack: "#000000",
     grey: "#404040",
-    darkGrey: "#fff",
-    lightGrey: "#C4C4C4",
+    darkGrey: "#6E7686",
```

Comment:
Should `darkGrey` and `lightGrey` be different..? If not, let's remove one.

Reply:
Hmm a little difficult since darkGrey is used for the sidebar background as of now... darkGrey would only be used as the background for the sidebar, so we should keep this until we figure out what the color of the side bar would be!